### ISSUE TITLE
Runs w/o logging in.

### ISSUE THREAD

**Issue by JustinMDP** [konkers/ff7run#56]
Invenerable has requested multiple times to allow for runs to be done without logging in. There may be some folk who are somewhat hesitant to do so for various reason. There may be some validity to offering a non-log in option, even if it's more limited as the tracker requires the functionality to persist between sessions.

**Comment by konkers** [comment]
This would require some pretty significant re-work to how the application is structured.

**Comment by JustinMDP** [comment]
That is fair. I just figured it was worth mentioning since if he's feeling awkward about having to log in, somebody else is bound to as well. Personally, however, I don't see the big deal.

**Comment by AKWolff037** [comment]
What about creating an "anonymous" login that generates a GUID and then use that GUID as the user key, and accept in the URL so they could just bookmark that run.  Would be single-run only

**Comment by konkers** [comment]
That could work.  It would require a conditional on the backend to handle not having auth  (see https://github.com/konkers/ff7run/blob/master/functions/src/index.ts#L41).  We'd probably want to namespace the anonymous runs so that you couldn't perform a guessing attack against the authenticated runs.

**Comment by AKWolff037** [comment]
Yeah, should be easy enough to do `/runs/anonymous/abcd-dehf-asjdk-234` or something

**Comment by konkers** [comment]
We can also throw the GUID into browser local storage so they user does not have to remember the URL.

**Comment by AKWolff037** [comment]
that dies if they clear cookies though right ?  Not a bad idea tho
